hr_timesheet_activity_begin_end/models/account_analytic_line.py

```diff
@@ -30,7 +30,7 @@ def _check_time_start_stop(self):
                     )
                 )
             hours = (stop - start).seconds / 3600
-            if hours and float_compare(hours, line.unit_amount, precision_digits=4):
+            if hours and float_compare(hours, line.unit_amount, precision_digits=2):
```

Comment:
Why you don't use precision configured in Odoo?

Reply:
What do you mean? There is no decimal.precision record for time i think? Anyhow I just modified what was already there

Reply:
I mean that literally, see `line.product_uom_id` and `uom_hour = self.env.ref('uom.product_uom_hour')` and https://github.com/odoo/odoo/blob/9ff767ec648bef997a3476117a64ceda75685843/addons/uom/models/uom_uom.py#L48